components/library/openssl/openssl-1.0.2/Makefile

```diff
@@ -199,14 +199,8 @@ install:	$(INSTALL_32_and_64) $(PROTO_DIR)/.openssl-0.9.8
 # We need to modify the default lint flags to include patched opensslconf.h from
 # the build directory. If we do not do that, lint will complain about md2.h
 # which is not enabled by default but it is in our opensslconf.h.
-LFLAGS_32 := -I$(BUILD_DIR_32)/include $(LINT_FLAGS) -lsoftcrypto
-LFLAGS_64 := -I$(BUILD_DIR_64)/include $(LINT_FLAGS) -lsoftcrypto
-
-# Set modified lint flags for our lint library targets.
-$(BUILD_DIR_32)/llib-lcrypto.ln: LINT_FLAGS=$(LFLAGS_32)
-$(BUILD_DIR_32)/llib-lssl.ln: LINT_FLAGS=$(LFLAGS_32)
-$(BUILD_DIR_64)/llib-lcrypto.ln: LINT_FLAGS=$(LFLAGS_64)
-$(BUILD_DIR_64)/llib-lssl.ln: LINT_FLAGS=$(LFLAGS_64)
+LFLAGS_32 := -I$(BUILD_DIR_32)/include -lsoftcrypto
+LFLAGS_64 := -I$(BUILD_DIR_64)/include -lsoftcrypto
```

Comment:
These variables does not seem useful anymore (and the comment above as well).

Reply:
Yes, I already did it but wasn't sure about it.
So you know the purpose of those .ln files?

Reply:
Maybe it would be better to create another PR with these changes because they don't seem to be related to the removal of the lint-libraries. What do you think?

Reply:
Remove either way you like.

I don't know the purpose of those .ln files, but they are clearly related to lint, hence up for removat.

Reply:
It seems, LDFLAGS_32 and LDFLAGS_64 here are only used to pacify lint and should be removed to avoid confusion.

Reply:
I am not sure whether the comment above should completely removed or only the lint part.
The comment says that we use the install code only for man installation which is 32 bit only.
64 bit is only for lint. So in my understanding we could also omit 64 bit install.
What do you think?